<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HostEditor</class>
 <widget class="QWidget" name="HostEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SwHost</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>551</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="main">
    <attribute name="title">
     <string>Main</string>
    </attribute>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>571</width>
       <height>441</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="reload">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>400</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>reload</string>
      </property>
     </widget>
     <widget class="QPushButton" name="enable">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>400</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;remove comment on selected.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>enable</string>
      </property>
     </widget>
     <widget class="QPushButton" name="disable">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>400</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;add comment on selected.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>disable</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>361</width>
        <height>351</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
     </widget>
     <widget class="QPushButton" name="addip">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>10</y>
        <width>51</width>
        <height>27</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;add a new ip radio button for batch edit&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>add ip</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>51</width>
        <height>27</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>add row to table</string>
      </property>
      <property name="text">
       <string>add</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="save2host">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>save to system hosts</string>
      </property>
      <property name="text">
       <string>save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveAs">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>91</width>
        <height>27</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;save quick swith hosts file for use in above, &lt;br/&gt;only can save 5, if already 5 exists, must del one or more first.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>saveAs ...</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="checkboxList">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>40</y>
        <width>161</width>
        <height>351</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>159</width>
         <height>349</height>
        </rect>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>161</width>
          <height>131</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="radioLayout"/>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="delhost">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>400</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>delete</string>
      </property>
     </widget>
    </widget>
    <widget class="QScrollArea" name="hostBtnList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>459</width>
        <height>29</height>
       </rect>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>160</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="btnLayout"/>
      </widget>
     </widget>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>10</y>
       <width>101</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="delswhost">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>10</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>del</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="edit">
    <attribute name="title">
     <string>Edit</string>
    </attribute>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>678</width>
       <height>471</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="savetxt">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>480</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="reloadtxt">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>480</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>reload</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="backup">
    <attribute name="title">
     <string>Backup</string>
    </attribute>
    <widget class="QListWidget" name="bakHostListWidget">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>120</y>
       <width>201</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="restorebakhost">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>80</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>restore</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delbakhost">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>delete</string>
     </property>
    </widget>
    <widget class="QLabel" name="bakuplist_label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Backup List :</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
